pallets/creditcoin/src/tests.rs

```diff
@@ -3275,11 +3275,43 @@ fn set_burn_gate_contract_fails_with_non_root() {
 fn set_burn_gate_faucet_address_fails_with_non_root() {
 	ExtBuilder::default().build_and_execute(|| {
 		let acct: AccountId = AccountId::new([0; 32]);
-		let addr: AccountId = AccountId::new([0; 32]);
 
 		assert_noop!(
-			Creditcoin::set_burn_gate_faucet_address(Origin::signed(acct), addr),
+			Creditcoin::set_burn_gate_faucet_address(Origin::signed(acct.clone()), acct),
 			BadOrigin
 		);
 	});
 }
+
+#[test]
+fn set_burn_gate_contract_passes_and_storage_is_updated() {
+	ExtBuilder::default().build_and_execute(|| {
+		let fake_address =
+			sp_core::H160([1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]);
+
+		let gate_contract =
+			DeployedContract { address: fake_address, chain: Blockchain::Luniverse };
+
```

Comment:
```suggestion

           assert!(Creditcoin::gate_contract().is_none());
```

formatting may be wrong though.

Reply:
Rustfmt likes the two line syntax